app/component/itinerary/WalkLeg.js

```diff
@@ -98,6 +98,13 @@ function WalkLeg(
       step?.feature?.wheelchairAccessible,
   )?.feature?.wheelchairAccessible;
 
+  // do not render subway exit/entrance if transfer happens within a station
+  const hideSubwayEntrances =
+    previousLeg?.mode === 'SUBWAY' &&
+    nextLeg?.mode === 'SUBWAY' &&
+    previousLeg.to.stop.parentStation?.gtfsId ===
+      nextLeg.from.stop.parentStation?.gtfsId;
```

Comment:
Can you create an `indoorUtils.js` file in the utils folder. Add this as a method with a name like `subwayTransferWithinSameStation(previousLeg, nextLeg)` I will add to this file in the future.

```
const hideSubwayEntrances = subwayTransferUsesSameStation(previousLeg, nextLeg);
```